crates/holochain-conductor-runtime-ffi/src/runtime.rs

```diff
@@ -112,6 +112,31 @@ impl RuntimeFfi {
         })
     }
 
+    /// Setup app
+    pub async fn setup_app(
+        &self,
+        payload: InstallAppPayloadFfi,
```

Comment:
This is maybe ignorant, but why is the installed app id in this payload optional, if calling it with `None` errors?

Reply:
No that's fair, updated the InstalledAppPayloadFfi type to require installed_app_id until the workaround can be removed: https://github.com/holochain/android-service-runtime/pull/87/commits/6fca789eeef0e6c4cc04a39c216d57fa07e92123